CHANGELOG.md

```diff
@@ -7,12 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- feat: add glob config provider [#713]
+- feat(build): enable glob config provider [#713]
```

Comment:
not sure if build should be exposed on changelog

Reply:
I guess from the user perspective, this is a single change.

Reply:
I mean

```suggestion
```